public-api/src/main/java/org/pmiops/workbench/cdr/dao/QuestionConceptDao.java

```diff
@@ -8,9 +8,10 @@
 public interface QuestionConceptDao extends CrudRepository<QuestionConcept, Long> {
 
     @Query(nativeQuery=true, value="SELECT c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence from\n" +
-            "concept c join concept_relationship r on c.concept_id = r.concept_id_2\n" +
+            "concept c \n" +
             "join achilles_results ar on ar.stratum_2=c.concept_id\n" +
-            "where r.concept_id_1=?1 and r.relationship_id = 'Module of' and ar.analysis_id=3110\n" +
-            "group by c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence")
-    List<QuestionConcept> findSurveyQuestions(long survey_concept_id);
+            "where ar.stratum_1=?1 and ar.analysis_id=3110\n" +
+            "group by c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence,ar.stratum_5 \n" +
```

Comment:
What is ar.stratum_5 ? In achilles_analysis  for 3110 it is defined as null.

Reply:
i used 3110 to have the question order in there so that questions fetched from this analysis can be fetched in order.